source/client-side-encryption/etc/data/encryptedFields-Range-Date.json

```diff
@@ -1,7 +1,4 @@
 {
-    "escCollection": "enxcol_.default.esc",
-    "eccCollection": "enxcol_.default.ecc",
-    "ecocCollection": "enxcol_.default.ecoc",
```

Comment:
@kevinAlbs: I just noticed this, but if you're removing the metadata collections here, doesn't that make this file identical to its `range-encryptedFields-<type>.json` counterpart used by [Prose Test 22](https://github.com/mongodb/specifications/blob/master/source/client-side-encryption/tests/README.rst#range-explicit-encryption)?

If so, would it make sense to just stick with a single file and update the `yamlfile()` calls in the test templates?

Reply:
Related PR where I encountered this: https://github.com/mongodb/specifications/pull/1399

Reply:
> The only difference I see is that the range- files also specify contention: 0, which is the default and may be irrelevant.

Another difference: the field name in [encryptedFields-Range-Decimal.json is encryptedDecimal](https://github.com/mongodb/specifications/blob/master/source/client-side-encryption/etc/data/encryptedFields-Range-Decimal.json#L13).

I suggest filing a DRIVERS ticket for the improvement. This change can be made in a separate PR to avoid adding unrelated changes to this PR.

Reply:
Opened https://jira.mongodb.org/browse/DRIVERS-2604.